.github/workflows/kind-dpu-offload.yml

```
@@ -0,0 +1,129 @@
+name: DPU Offload
+
+on:
+  pull_request:
+    branches: [ master, release-* ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)